src/test/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasLineAndPointSlicingTest.java

```diff
@@ -0,0 +1,453 @@
+package org.openstreetmap.atlas.geography.atlas.raw.slicing;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.items.Line;
+import org.openstreetmap.atlas.geography.atlas.items.Point;
+import org.openstreetmap.atlas.geography.atlas.pbf.slicing.identifier.CountrySlicingIdentifierFactory;
+import org.openstreetmap.atlas.geography.atlas.raw.creation.RawAtlasGenerator;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.locale.IsoCountry;
+import org.openstreetmap.atlas.streaming.compression.Decompressor;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.resource.InputStreamResource;
+import org.openstreetmap.atlas.tags.ISOCountryTag;
+import org.openstreetmap.atlas.tags.SyntheticBoundaryNodeTag;
+import org.openstreetmap.atlas.tags.SyntheticNearestNeighborCountryCodeTag;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+
+/**
+ * {@link RawAtlasCountrySlicer} unit tests for slicing Lines and Points.
+ *
+ * @author mgostintsev
+ */
+public class RawAtlasLineAndPointSlicingTest
+{
+    private static CountryBoundaryMap COUNTRY_BOUNDARY_MAP;
+    private static Set<IsoCountry> COUNTRIES;
+
+    static
+    {
+        COUNTRIES = new HashSet<>();
+        COUNTRIES.add(IsoCountry.forCountryCode("CIV").get());
+        COUNTRIES.add(IsoCountry.forCountryCode("GIN").get());
+        COUNTRIES.add(IsoCountry.forCountryCode("LBR").get());
+    }
+
+    @Rule
+    public LineAndPointSlicingTestRule setup = new LineAndPointSlicingTestRule();
+
+    @BeforeClass
+    public static void setup()
+    {
+        COUNTRY_BOUNDARY_MAP = new CountryBoundaryMap(
+                new InputStreamResource(RawAtlasLineAndPointSlicingTest.class
```

Comment:
When creating an `InputStreamResource` it is usually better to provide a supplier here instead of the stream directly, so the resource can be reused. It might not impact here, but just in case.